docs/proposals/compute-resource-metrics-api.md

```diff
@@ -74,17 +83,20 @@ monitoring-heapster-abcde  0.12 cores  302 MB
 kube-ui-v1-nd7in           0.07 cores  130 MB
 ```
 
-A second user will be the scheduler. To assign pods to nodes efficiently, the
-scheduler needs to know the current free resources on each node.
+#### Scheduler
+
+Node usage data will be used for scheduling best-effort pods.
+To provide QoS support, the schduler will use historical usage of existing pods.
```

Comment:
Can you elaborate on why historical use of existing pods is needed to provide qos support?